roles/crawler/tasks/configure.yml

```diff
@@ -38,6 +38,12 @@
     # to pull, so check the boolean var for toggling cloning functionality.
     when: fpsd_crawler_clone_git_repo == true
 
+  - name: Populate config.ini.
+    template:
+      src: config.ini.j2
+      dest: "{{ fpsd_crawler_project_directory }}/fpsd/config.ini"
```

Comment:
1. I find it convenient to have it placed under the Virtualbox shared directory.
2. Aside from the systemd unit placement, all the code for our application is placed under `/opt/fingerprint-securedrop` as is all our own logging. This is how applications under `/opt` generally operate, i.e., they are mostly self-contained in their directory.